.travis.yml

```diff
@@ -4,7 +4,6 @@ node_js:
   - "0.11"
   - "0.10"
   - "0.8"
-  - "0.6"
```

Comment:
0.6 had to be removed because it's missing some functionality. But it's super dooper old, so the minimum requirement is node 0.8 (which is still pretty old)